.github/workflows/_publish-internal.yml

```diff
@@ -21,22 +17,22 @@ on:
                 description: "Choose the package to publish"
                 type: choice
                 options:
-                -   "dbt-adapters"
                 -   "dbt-athena"
                 -   "dbt-bigquery"
                 -   "dbt-postgres"
                 -   "dbt-redshift"
                 -   "dbt-snowflake"
                 -   "dbt-spark"
-            deploy-to:
-                description: "Choose whether to publish to test or prod"
-                type: environment
-                default: "test"
             branch:
                 description: "Choose the branch to publish"
                 type: string
                 default: "main"
 
+# don't publish to the same target in parallel
```

Comment:
This concurrency should be here, instead of copy/pasted in multiple downstream callers.